app/src/main/java/de/danoeh/antennapod/fragment/FeedInfoFragment.java

```diff
@@ -218,18 +235,44 @@ private void showFeed() {
             lblAuthor.setVisibility(View.GONE);
             txtvAuthor.setVisibility(View.GONE);
         }
-        if (!TextUtils.isEmpty(feed.getLanguage())) {
-            txtvLanguage.setText(LangUtils.getLanguageString(feed.getLanguage()));
-        } else {
-            lblLanguage.setVisibility(View.GONE);
-            txtvLanguage.setVisibility(View.GONE);
-        }
+
         txtvUrl.setText(feed.getDownload_url() + " {fa-paperclip}");
         Iconify.addIcons(txtvUrl);
-
         getActivity().invalidateOptionsMenu();
     }
 
+    private void loadStatistics() {
+        if (disposable != null) {
+            disposable.dispose();
+        }
+
+        disposable =
+                Observable.fromCallable(() -> {
+                    StatisticsItem statisticsData = DBReader.getStatisticsFromId(feed.getId());
+                    return statisticsData;
+                })
+                        .subscribeOn(Schedulers.io())
+                        .observeOn(AndroidSchedulers.mainThread())
+                        .subscribe(result -> {
+                            podcastStatistics = result;
+                            if (podcastStatistics != null) {
+                                txtvPodcastTime.setText(Converter.shortLocalizedDuration(
+                                        getContext(), podcastStatistics.timePlayed));
+                                txtvPodcastSpace.setText(Formatter.formatShortFileSize(
+                                        getContext(), podcastStatistics.totalDownloadSize)
+                                        + " • "
```

Comment:
I like the idea, but actually I'd keep it consistent with the other statistics screens (thus: stick to two lines). Especially because this section will be displayed above the description (which otherwise on small screens might not be visible above the fold).

Reply:
> I like the idea, but actually I'd keep it consistent with the other statistics screens (thus: stick to two lines).

@keunes Having it in only 2 (instead of 3) lines looks a bit strange, in my opinion. Have a look at the screenshots below.

<img width="200" src="https://user-images.githubusercontent.com/5811634/97235739-d2685500-17e3-11eb-83b5-1eaeee87b26f.png" /> <img width="200" src="https://user-images.githubusercontent.com/5811634/97235743-d4321880-17e3-11eb-88f1-6e4e201b323c.png" />

> Especially because this section will be displayed above the description

Maybe we could even remove the authors section from that screen because the authors are displayed in the blurred area anyway. When doing that (and making headings the same font size+color as on the "add podcast" screen), the screen has more space for the description and looks more clean. See screenshot below.

<img width="200" src="https://user-images.githubusercontent.com/5811634/97236562-a352e300-17e5-11eb-9343-f8db22f4f78d.png" />

Reply:
I agree with @ByteHamster,  In my opinion 3 rows is looking clean than having 2 rows. Also it is looking good after removing Author section.

Reply:
Let me know if I need to make any changes and push again.

Reply:
@ByteHamster Can you share a screenshot of https://feeds.soundcloud.com/users/soundcloud:users:216894809/sounds.rss on a 5 inch 1080×1920 px HD 446 ppi display? My main concern is that the 'Description' title will be below the fold - if that's secured also on smaller screens (or even smaller than my screen).

Reply:
> and making headings the same font size+color as on the "add podcast" screen

What do you mean?